### ISSUE TITLE
struct implementation

### ISSUE THREAD

**Issue by shamin1998** [rahulkumaran/sudocode#28]
**Is your feature request related to a problem? Please describe.**
Structures are a very useful tool in coding, and lack of support for it would be a huge handicap for users.

**Describe the solution you'd like**
We could come up with a simple structure pseudocode syntax, using which users could define their own structures, members and all.

**Additional context**
This could also easily be used for typedef implementation, another huge tool for users.

**Comment by rahulkumaran** [comment]
But will people actually define a struct in pseudocode? @shamin1998

**Comment by shamin1998** [comment]
I think majority of algorithms are implemented using structures only, even though it's not explicitly stated so.
For example, BST implementation in Cormen, uses T.root, x.left, etc. 

I don't see how one would be able to apply these pseudocodes without structures.

**Comment by rahulkumaran** [comment]
Right! Cool. Makes sense. You can start working on this if you wish.
